src/lib/CONFIG/config.cpp

```diff
@@ -302,6 +341,8 @@ TxConfig::SetDefaults()
         SetBoostChannel(0);
         SetSwitchMode((uint8_t)smHybrid);
         SetModelMatch(false);
+        SetFanMode(0);
+        SetMotionMode(0);
```

Comment:
Should these be protected by `#ifdef HAS_GSENSOR`??

Reply:
I think we should leave the function and and guard the body so we don't have guards all over the place.